src/app/ApplicationTemplate.Tests/Business/UserProfileServiceShould.Mock.cs

```diff
@@ -0,0 +1,14 @@
+﻿using ApplicationTemplate.Client;
+
+namespace ApplicationTemplate.Tests.Business;
+
+public partial class UserProfileServiceShould
```

Comment:
You can use the `sealed` keyword for classes that aren't inherited it will make the application more performant in the future when it's bigger and everyone uses it.